src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -154,6 +158,9 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
       this.retryBackoffMs = config.retryBackoffMs();
       this.maxRetries = config.maxRetries();
       this.timeout = config.readTimeoutMs();
+      String tasksMax = props.get(TASKS_MAX_CONFIG);
+      this.retryOnConflict = (tasksMax == null) ? TASKS_MAX_DEFAULT : Integer.parseInt(tasksMax);
```

Comment:
if we are trying to retry the same amount of times that there are concurrent requests, the safer number to pick here is max concurrent requests instead plus it is much cleaner

Reply:
Do you mean the `MAX_IN_FLIGHT_REQUESTS_CONFIG` ?